openshift/backend.template.yaml

```diff
@@ -81,8 +81,73 @@ objects:
         labels:
           app.kubernetes.io/name: tangerine-backend
       spec:
+        initContainers:
+          - name: db-migrations
+            image: ${IMAGE}:${IMAGE_TAG}
+            command: ["flask", "db", "upgrade"]
+            env:
+              - name: ENABLE_HYBRID_SEARCH
+                value: ${ENABLE_HYBRID_SEARCH}
+              - name: STORE_INTERACTIONS
+                value: ${STORE_INTERACTIONS}
+              - name: ENABLE_RERANKING
+                value: ${ENABLE_RERANKING}
+              - name: ENABLE_QUALITY_DETECTION
+                value: ${ENABLE_QUALITY_DETECTION}
```

Comment:
Technically, no need for these env vars on the init container

Reply:
Good catch. Fixed.